recipes/libsecret/all/conanfile.py

```diff
@@ -38,11 +51,21 @@ def config_options(self):
     def configure(self):
         if self.options.shared:
             del self.options.fPIC
+
+        if self.options.with_libgcrypt and self.settings.os != "Linux":
+            self.output.warn(
+                "gcrypt is only available on Linux. Option will be ignored."
+            )
+            del self.options.with_libgcrypt
```

Comment:
We should also delete so there's only less package_id that can be generated?

```suggestion
        if self.settings.os != "Linux":
            del self.options.with_libgcrypt
```